CommandFactory.py

```diff
@@ -0,0 +1,10 @@
+from CreateCommand import CreateCommand
+from ParseInput import parse_args
+args= parse_args()
+class CommandFactory:
+    @staticmethod
+    def build_command(args):
```

Comment:
You are not building the command. You are creating it. rename the funtion to create_command. 

There is another design pattern called `builder design pattern` which is different from Factory design pattern